gamer-os/packages.x86_64

```diff
@@ -0,0 +1,66 @@
+arch-install-scripts
```

Comment:
is there a way to also include the frzr AUR package? Either built from the AUR (preferred) or pre-built package which is currently available here: https://github.com/gamer-os/frzr/releases

Reply:
This can be done. Archiso uses the airootfs directory as base for its root filesystem of the live environment which you boot to when booting from the ISO it generates. The script root/customize_airootfs.sh is run before finalizing the ISO.

What you can do is adding the package to the airootfs directory and then change the customize_airootfs.sh script to install it using pacman. That way frzr will be available on the live environment booted from the ISO.

Another option would be to host frzr in a package repository which can be used by pacman. Then you can add your repo to the pacman.conf file and the package name to the packages.x86_64 file.